extract/tests/zalando/zalando_test.py

```diff
@@ -0,0 +1,50 @@
+from tests.test_utils import read_test_html
```

Comment:
same here: better use relative import: `../test_utils`. You need to create `__init__.py` files in `tests` and `tests/zalando`